e2e/cypress/integration/system_console/revoke_all_sessions.js

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import users from '../../fixtures/users.json';
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('SC17020 - Revoke All Sessions from System Console', () => {
+    it('Verify for System Admin', () => {
+        // # Login as System Admin
+        cy.apiLogin('sysadmin');
+
+        cy.visit('/admin_console/user_management/users');
+
+        // * Verify the presence of Revoke All Sessions button
+        cy.get('#revoke-all-users').should('be.visible').and('not.have.class', 'btn-danger').click();
+
+        // * Verify the confirmation message when users clicks on the Revoke All Sessions button
+        cy.get('#confirmModalLabel').should('be.visible').and('have.text', 'Revoke all sessions in the system');
+        cy.get('.modal-body').should('be.visible').and('have.text', 'This action revokes all sessions in the system. All users will be logged out from all devices. Are you sure you want to revoke all sessions?');
+        cy.get('#confirmModalButton').should('be.visible').and('have.class', 'btn-danger');
+
+        // # Click on Cancel button in the confirmation message
+        cy.get('.modal-footer .btn-cancel').click();
```

Comment:
Please add ID to cancel button.

Reply:
Hi @saturninoabril 

I have added the ID and have updated the test accordingly. Please recheck.